app/decorators/repository_decorator.rb

```diff
@@ -1,19 +1,38 @@
 # frozen_string_literal: true
 
 class RepositoryDecorator < ApplicationDecorator
+  include ActionView::Helpers::NumberHelper
+
   delegate_all
 
   def self.collection_decorator_class
     PaginatingDecorator
   end
 
   def languages
-    return '' unless language
+    return [] unless language
 
-    language.split(',').first(3).to_sentence
+    language.split(', ').first(3).map(&:strip)
   end
 
   def name
     link[/[\w_-]+$/]
   end
+
+  def issues_formatted
+    format_number(model.issues)
+  end
+
+  def stars_formatted
+    format_number(model.stars)
+  end
+
+  private
+
+  def format_number(number)
+    return '0' if number.nil?
+    return number.to_s if number <= 999
```

Comment:
NIT: What if we set `0` as default?

```suggestion
  def format_number(number = 0)
    return number.to_s if number <= 999
```

Reply:
Nice tip! Thanks!